docs/DeveloperGuide.md

```diff
@@ -649,14 +676,14 @@ To ensure that code coverage reporting includes both general unit tests and UI t
 
 1. A new Gradle task `uiTest` was created to only run UI tests that are unit test files that end with `UiTest`
 2. The default `test` task is configured to exclude such files
-3. The `jacocoTestReport` task is modified to only depend on (i.e. run before) the `uiTest` task is the system's OS is not MacOS, Ubuntu or *nux (i.e. Windows only)
+3. The `jacocoTestReport` task is modified to only depend on (i.e. run before) the `uiTest` task if the system's OS is not MacOS, Ubuntu or *nux (i.e. Windows only)
```

Comment:
Just checking is the *nux an actual thing or a typo? Never head of it but if its a thing then its fine

Reply:
Oh no, it's part of the clause I used:

```groovy
    def disallowedOSes = ['mac', 'ubuntu', 'nux']
```

Reply:
<img width="1035" alt="Screenshot 2023-11-13 at 5 07 08 PM" src="https://github.com/AY2324S1-CS2103-T16-3/tp/assets/53000252/46c48192-e9eb-4a80-a7ae-e173ea83cd4b">
So just to confirm it's supposed to show up like this right